.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI tests
+
+on: [push, pull_request]
+
+jobs:
+
+  Run-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8']
+    steps:
+
+    - uses: actions/checkout@v1
+      with:
+        submodules: recursive
+
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install
+      run: |
+        sudo apt-get update
+        sudo apt-get install clang-format-5.0 xsltproc git
```

Comment:
Why `clang-format-5.0`?

Reply:
This yaml is a copy/paste, so this is an unneeded carry over.